iothub/service/src/Http/HttpClientFactory.cs

```diff
@@ -30,7 +30,7 @@ internal class HttpClientFactory
         //
         // This default value is consistent with the default value used in Azure.Core
         // https://github.com/Azure/azure-sdk-for-net/blob/7e3cf643977591e9041f4c628fd4d28237398e0b/sdk/core/Azure.Core/src/Pipeline/ServicePointHelpers.cs#L29
-        private static readonly TimeSpan DefaultConnectionLeaseTimeout = TimeSpan.FromMinutes(5);
+        private static readonly TimeSpan s_defaultConnectionLeaseTimeout = TimeSpan.FromMinutes(5);
```

Comment:
Tim, this is still used in service point manager.

Reply:
Yes, that is by design. We use the service point manager to set this timeout so that users avoid problems around failover scenarios. Did you want to change this in some way?